library/threading.c

```diff
@@ -35,12 +36,17 @@ static void threading_mutex_init_pthread( mbedtls_threading_mutex_t *mutex )
     if( mutex == NULL )
         return;
 
-    mutex->is_valid = pthread_mutex_init( &mutex->mutex, NULL ) == 0;
+    mutex->is_valid = (pthread_mutex_init( &mutex->mutex, NULL ) == 0) ? 1 : 0;
+}
+
+static bool is_mutex_valid(mbedtls_threading_mutex_t *mutex)
+{
+    return (mutex->is_valid == 1);
```

Comment:
style: add space after `(` and before `)`

Reply:
Fixing.